be/src/common/signal_handler.h

```diff
@@ -254,6 +256,9 @@ void DumpTimeInfo() {
     formatter.AppendString(" try \"date -d @");
     formatter.AppendUint64(static_cast<uint64>(time_in_sec), 10);
     formatter.AppendString("\" if you are using GNU date ***\n");
+    formatter.AppendString("*** Current BE version: ");
```

Comment:
Do not use this method. These code will allocate memory and string is very complicated. std::string get_build_version(bool compact) {
    std::stringstream ss;
    ss << DORIS_BUILD_VERSION
#ifdef NDEBUG
       << " RELEASE"
#else
       << " DEBUG"
#endif
       << " (build " << DORIS_BUILD_HASH << ")";

    if (!compact) {
        ss << std::endl << "Built on " << DORIS_BUILD_TIME << " by " << DORIS_BUILD_INFO;
    }

    return ss.str();
}

You could just use the macro